instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -61,24 +61,31 @@ def hello():
 from opentelemetry import context, propagators, trace
 from opentelemetry.instrumentation.utils import http_status_to_canonical_code
 from opentelemetry.instrumentation.wsgi.version import __version__
+from opentelemetry.trace.propagation.textmap import DictGetter
 from opentelemetry.trace.status import Status, StatusCanonicalCode
 
 _HTTP_VERSION_PREFIX = "HTTP/"
 
 
-def get_header_from_environ(
-    environ: dict, header_name: str
-) -> typing.List[str]:
-    """Retrieve a HTTP header value from the PEP3333-conforming WSGI environ.
+# pylint:disable=arguments-differ
```

Comment:
Is this because we renamed carrier to environ and key to header_name in the subclass? If so, why not use carrier and key in the first place? It's anyway safer in case something calls the getter with named arguments.

Reply:
the initial get function which was used in WSGI had these argument names, I consider changing it to carrier and key but this made more contextual sense.

Reply:
I think we should stick to argument names specified by the getter interface. I feel using different names isn't adding enough value to justify not satisfying the interface 100%. We can explain in doc string that carrier is WSGI environ and key is header name.

Reply:
@nprajilesh how do you think we should handle this?

Reply:
@owais yah, I think you are right. I've updated the argument names and added docstring for clarity.